aslam_cv_common/cmake/export_flags.cmake

```diff
@@ -1,14 +1,9 @@
-set(IS_ARM_ARCHITECTURE FALSE)
-execute_process(COMMAND uname -m COMMAND tr -d '\n' OUTPUT_VARIABLE ARCHITECTURE)
-if (ARCHITECTURE MATCHES "^(arm)")
-  set(IS_ARM_ARCHITECTURE TRUE)
-endif()
-
-if (NOT ANDROID AND NOT IS_ARM_ARCHITECTURE)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
-elseif (IS_ARM_ARCHITECTURE)
-  message(STATUS "Setting ARM compilation flags.")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfpu=neon -march=armv7-a")
+add_definitions(-std=c++11)
+execute_process(COMMAND uname -m COMMAND tr -d '\n' OUTPUT_VARIABLE ARCH)
+if (ARCH MATCHES "^(arm)")
+  add_definitions(-mfpu=neon)
```

Comment:
The 3rdparty dependencies will not get these flags. We might also need to inject these settings using a catkin profile.